jovian/jovian_nb_ext/main.js

```diff
@@ -3,7 +3,7 @@ define([
   "base/js/namespace",
   "base/js/dialog",
   "base/js/keyboard"
-], function($, Jupyter, dialog, keyboard) {
+], function ($, Jupyter, dialog, keyboard) {
```

Comment:
Install `Prettier - code formatter` vscode extension. to avoid these changes

Reply:
Hey @RogodaThallus install this extension for the next commits. These are not code changes, we want to keep the format consistent.

Reply:
It is installed and was used prior to last commit. It should be fine now.